mermaid/example.ts

```diff
@@ -0,0 +1,17 @@
+import { DirectedGraph } from "../graph/DirectedGraph.ts";
```

Comment:
I think we don't want to commit this file, though parts of it should be relocated to a `readme.md` entry.